HpToolsLauncher/Launcher.cs

```diff
@@ -464,6 +465,13 @@ private IAssetRunner CreateRunner(bool isFirstRun, List<TestInfo> reruntests = n
                     }
                     ConsoleWriter.WriteLine("Launcher timeout is " + timeout.ToString(@"dd\:\:hh\:mm\:ss"));
 
+                    bool leaveUftVisibleIfOpen = false;
+                    string leaveUftOpenIfVisibleParam = _ciParams.GetOrDefault("LEAVE_UFT_OPEN_IF_VISIBLE", "0");
+                    if (_ciParams.ContainsKey("LEAVE_UFT_OPEN_IF_VISIBLE") && leaveUftOpenIfVisibleParam.Equals("1"))
+                    {
+                            leaveUftVisibleIfOpen = true;
+                    }
+

```

Comment:
I would suggest to replace the new block (468-474) with this line:
bool leaveUftOpenIfVisible = _ciParams.GetOrDefault("LEAVE_UFT_OPEN_IF_VISIBLE", "0") == "1";
(I also renamed leaveUftVisibleIfOpen as leaveUftOpenIfVisible).
Please test.